src/interface/inference.py

```diff
@@ -0,0 +1,180 @@
+'''Run a REPL or corpus-level inference
```

Comment:
Add a bit more in the way of usage instructions.

Reply:
Make it clear this is CoLA only, and add some notes on what it'd take to make this flexible, or else just do the work to make it flexible.

Reply:
Also spell out what a REPL is.